covid_act_now/delphi_covid_act_now/run.py

```diff
@@ -83,6 +83,7 @@ def run_module(params):
             sensor=SIGNALS[1])
 
         earliest, latest = min(exported_csv_dates), max(exported_csv_dates)
+        files_exported = df.size * 2  # multiply by two to count both positivity and tests signals
```

Comment:
```suggestion
        files_exported = exported_csv_dates.size * 2  # multiply by two to count both positivity and tests signals
```

we want the number of files, not the number of lines

Reply:
ah right i forget we have multiple geo values. should be the sum of all lines across all geos right? 

also i wonder whats going on with that comment formatting...